src/julia.h

```diff
@@ -521,7 +521,6 @@ extern JL_DLLEXPORT jl_value_t *jl_stackovf_exception;
 extern JL_DLLEXPORT jl_value_t *jl_memory_exception;
 extern JL_DLLEXPORT jl_value_t *jl_readonlymemory_exception;
 extern JL_DLLEXPORT jl_value_t *jl_diverror_exception;
-extern JL_DLLEXPORT jl_value_t *jl_domain_exception;
 extern JL_DLLEXPORT jl_value_t *jl_overflow_exception;
 extern JL_DLLEXPORT jl_value_t *jl_inexact_exception;
```

Comment:
Yes, looks like you're right. There are no uses of `cvt_iintrinsic_checked` after this, so even though it looks like there is another use of `jl_inexact_exception` in runtime_intrinsics, it seems to be dead code.